### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nikodem-stolarczyk**
drafted the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
-# spectral-line-shapes
\ No newline at end of file
+# spectral-line-shapes
+
+The spectral-line-shapes repository hosts the implementations of the modified Hartman-Tran (`mHT`) spectral line-shape profile. The mHT is a variation of the Hartman-Tran profile (HT or HTP, introduced in 10.1016/j.jqsrt.2013.06.015). The article describing the mHT spectral lineshape function is currently under preparation.
+
+The spectral-line-shapes repository features the implementations of the complex problability function (cpf), which are used to calculate the mHT. Though optimized specifically to work with the mHT, the cpf functions can also be used separately. The repository features two cpf functions: `cpf_accurate` (maintaining high accuracy, based on highly-modified 64-term Weideman algorithm jstor.org/stable/2158232), and `cpf_fast` (faster but less accurate, utilizing Humlicek's aproach in its first suberegion, 10.1016/0022-4073(82)90078-4, and 24-term Weideman approach elsewhere jstor.org/stable/2158232). The detailed description of the modifications of the cpf functions will be summarized in the upcoming article.
```
Comment (hjozwiak-umk) [inline README.md]: "suberegion" -> "subregion"

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+```
+ 
+## Dependencies
+
+Depending on the implementation, the following programs are required
+
+### python
+- Python 3.X
+### fortran
+
```
Comment (hjozwiak-umk) [inline README.md]: * [Make](https://www.gnu.org/software/make/)
* A modern Fortran compiler, such as [GNU Fortran Compiler](https://gcc.gnu.org/wiki/GFortran) (gfortran) or
 [Intel Fortran Compiler](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/fortran-compiler.html) (ifort)

**Comment by hjozwiak-umk** [comment]
We could add the output of usage_example and the plots to the Readme file to show the expected outcome of a test run @nikodem-stolarczyk

**Comment by nikodem-stolarczyk** [comment]
@hjozwiak-umk I've just updated the python readme. Could you add the plots?

**Pull request merged**